Makefile

```
@@ -1,12 +1,22 @@
 GOMINVERSION = 1.16
 NEBULA_CMD_PATH = "./cmd/nebula"
-BUILD_NUMBER ?= dev+$(shell date -u '+%Y%m%d%H%M%S')
+NEBULA_CMD_SUFFIX =
+BUILD_NUMBER ?= $(shell git describe --abbrev=0 --match "v*" | cut -dv -f2)-$(shell git branch --show-current)-$(shell git describe --dirty | cut -d- -f2-)
```

```suggestion
BUILD_NUMBER ?= $(shell git describe --abbrev=0 --match "v*" | cut -dv -f2)-$(shell git branch --show-current)-$(shell git describe --long --dirty | cut -d- -f2-)
```

`--long` on the last describe makes it work better when you are building on the same git commit with the tag